packages/fx-core/src/component/debugHandler/appManifest.ts

```diff
@@ -174,6 +178,21 @@ export class AppManifestDebugHandler {
       // For SPFx manifest
       this.envInfoV3.config.isLocalDebug = true;
 
+      // Local debug if switching to a different account in same tenant
+      if (
+        this.envInfoV3.envName === environmentManager.getLocalEnvName() &&
```

Comment:
No need to check the envName because only local debug will enter here.

Reply:
updated.